install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -208,6 +208,13 @@ containers:
   - name: ISTIO_META_SDS_TOKEN_PATH
     value: "{{ .Values.global.sds.customTokenDirectory -}}/sdstoken"
   {{- end }}
+  {{- if .Values.global.meshID }}
```

Comment:
Don't forget istio/installer - operator and some profiles will use that.

Reply:
Yes, will follow.